rootfs/scheduler/__init__.py

```diff
@@ -190,7 +43,7 @@
 
 SECRET_TEMPLATE = """\
 kind: Secret
-apiVersion: $version
+apiVersion: v1
```

Comment:
Because deployments for example does not use v1 - the global variable buys us nothing and should rather be treated per resource object